functions.sh

```diff
@@ -124,20 +131,18 @@ copy_interesting_files_from_system() {
 }
 
 validate_RESULT() {
-    disksdir=$1
```

Comment:
I think having it as a local named variable might have been useful, but never mind.

Reply:
+1 it's more readable to set a name for the variable.